web/html/src/utils/functions.test.ts

```diff
@@ -28,6 +28,18 @@ test("check order by date function'", () => {
   expect(arr[0]).toEqual(eRaw);
 });
 
+describe("capitalize", () => {
+  test("simple cases", () => {
+    expect(Utils.capitalize("foo")).toEqual("Foo");
+    expect(Utils.capitalize("foo-bar")).toEqual("Foo Bar");
+    expect(Utils.capitalize("fooBar")).toEqual("Foobar");
```

Comment:
If I may, I would actually expect `fooBar` to became `Foo Bar`, but it's just my feeling. Do you have instead any concrete example where your rule applies properly?

Reply:
`fooBar` converted to `Foobar` is the previous behavior... I didn't change that at all.

Reply:
Okay then, never mind it's not part of this PR. Thanks for clarifying :+1: